### PR TITLE
Views can specify the tag name

### PR DESCRIPTION
**Pull request by fao89**
https://pulp.plan.io/issues/6832
closes #6832

Please be sure you have read our documentation on creating PRs:
https://docs.pulpproject.org/contributing/pull-request-walkthrough.html

### CODE REVIEW

File: CHANGES/6832.feature [hunk 1]
```diff
@@ -0,0 +1 @@
+Views can specify the tag name with `pulp_tag_name`
```
Comment (daviddavis) [inline CHANGES/6832.feature:1]: Should this go into the plugin_api folder?

**Comment by daviddavis** [comment]
Can you add some docs for the plugin writer too?

File: CHANGES/6832.feature (hunk 1 above)
Comment (fao89) [reply CHANGES/6832.feature:1]: not sure

**Comment by fao89** [comment]
> Can you add some docs for the plugin writer too?

yes, I didn't add before because I thought it would go as a hidden feature on 3.4.0

File: CHANGES/6832.feature (hunk 1 above)
Comment (daviddavis) [reply CHANGES/6832.feature:1]: I think the main difference is that changes in `CHANGES/plugin_api` are for plugin writers while `CHANGES` are for endusers. If I understand this change correctly, it seems more geared toward plugin writers?

**Comment by daviddavis** [comment]
> yes, I didn't add before because I thought it would go as a hidden feature on 3.4.0

I think this'll probably land in 3.5.0.

File: CHANGES/6832.feature (hunk 1 above)
Comment (fao89) [reply CHANGES/6832.feature:1]: yep, it is for plugin writers, I'll move to `CHANGES/plugin_api` then

**Comment by fao89** [comment]
> Can you add some docs for the plugin writer too?

Is it a good place to add the docs?
https://docs.pulpproject.org/plugins/reference/how-to-doc-api.html

**Comment by daviddavis** [comment]
I'm not sure. This might also be a good place:

https://docs.pulpproject.org/plugins/plugin-writer/concepts/subclassing/viewsets.html

@bmbouter any thoughts?

**Pull request merged**